assessment.js

```diff
@@ -41,7 +67,10 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.update(
+  { house: "House Arryn"},
+  { $set: { motto:""}}
+)
 // your code ends here
 
```

Comment:
This doesn't work the way you expect it to...
Motto is still there and set as "As high as honor"